### PR TITLE
PHX-245 Update text to inform users they must submit their answer to each problem

### PR DESCRIPTION
**Pull request by douglashall**
This change is to improve the UX for in-flight courses in production.

Related PR:
https://github.com/edx/edx-platform/pull/11362

### CODE REVIEW

**Comment by douglashall** [comment]
@afzaledx @cahrens

File: edx_proctoring/templates/timed_exam/ready_to_submit.html [hunk 1]
```diff
@@ -7,7 +7,12 @@
   </h3>
   <p>
     {% blocktrans %}
-      Make sure your answers are ready to be submitted, and then submit your exam. Your exam will then be graded.
+      Make sure that you have selected "Check" or "Final Check" for each problem before you submit your exam.
+    {% endblocktrans %}
+  </p>
+  <p>
+    {% blocktrans %}
+      After you submit your exam, your exam will be graded.
```
Comment (cahrens) [inline edx_proctoring/templates/timed_exam/ready_to_submit.html]: Nit: each problem is graded at the time that you click "Final Check", isn't it? What does submitting the exam actually do?

**Comment by cahrens** [comment]
I feel somewhat uneasy about "After you submit your exam, your exam will be graded.", but the text certainly makes it clear that you have to click "Check" or "Final Check". :+1:

**Comment by cahrens** [comment]
@douglashall didn't we also want to put something in the bar that appears at the top of the courseware?

**Comment by douglashall** [comment]
@cahrens Yes, the header bar text is in edx-platform. Just tagged you on that PR as well. Thanks.

**Comment by cahrens** [comment]
Yep, I just saw it!

File: edx_proctoring/templates/timed_exam/ready_to_submit.html (hunk 1 above)
Comment (douglashall) [inline edx_proctoring/templates/timed_exam/ready_to_submit.html]: I'm not sure what the original intention of the "Your exam will then be graded" text was, but this is just a rewording of that original text. Submitting the exam causes a proctored/timed exam attempt to be recorded. Is it possible the grade wouldn't appear on the progress page until the exam is submitted? I can check that.

**Comment by afzaledx** [comment]
@douglashall, it might be a good idea to add these instructions to the "ready_to_start.html" template as well. This screen is shown once the students have entered the unique exam code in the proctoring app and verified their identity and are just about to enter the exam.

**Pull request merged**